### DISCUSSION TITLE
Op Video Texture Error: Could not load Video

### DISCUSSION THREAD

**Discussion by fwaveforms** [cables-gl/cables_docs#828] [Answers & Questions]
Hi, I just started using Cables a few days ago. It's the most straightforward tool of it's kind I have used so far, so I'm looking forward to really getting my head around it. 
I have had a look around on videos and documentation and haven't found any info on this error. 

I'm using the standalone version on an Intel Mac book pro. When I try to use a video texture I get:

"Error: Could not load video / DEMUXER_ERROR_COULD_NOT_OPEN: FFmpegDemuxer: open context failed"

I'm using .mp4 videos.

Does anyone have an idea of how to fix this?

**Comment by pandrr** [comment]
discussed here: cables-gl/cables_electron/issues/132
